package/README.md

```diff
@@ -12,7 +12,7 @@ To consume all components you must be running npm version 5 or above.
 
 ## Installation
 
-    npm install --save @govuk-frontend/all
+    npm install --save @govuk-frontend/govuk-frontend
```

Comment:
should this say `@govuk-frontend/govuk-frontend-placeholder`?

Reply:
it can say `magic-land-awaits`

Reply:
yeah it should.